<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileStatus</class>
 <widget class="QDialog" name="FileStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileStatus</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Folder:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_leFolder"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_btBrowser">
       <property name="text">
        <string>Browser...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="m_twFiles">
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="my_commentaries">
       <property name="text">
        <string>no commentary</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btAnalysis">
       <property name="text">
        <string>Analysis</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
